src/docs/concepts-users-and-authentication/users-and-authentication-in-flex/index.md

```diff
@@ -28,7 +28,8 @@ you will need to create those limitations in your client application.
 You can use [extended data](/concepts/extended-data-introduction/) to
 determine that a user is in a certain group, and then allow a subset of
 your marketplace functionalities, e.g. listing creation, for a specified
-group.
+group. You can, for example, define user types in Console for this
```

Comment:
I think linking to help center to the user types article here would be good as we don't have info on user types in docs